src/libcore/cell.rs

```diff
@@ -368,16 +367,6 @@ impl<T: PartialEq> PartialEq for RefCell<T> {
     }
 }
 
-#[unstable]
-impl<T:fmt::Show> fmt::Show for RefCell<T> {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self.try_borrow() {
-            Some(val) => write!(f, "{}", val),
-            None => write!(f, "<borrowed RefCell>")
-        }
-    }
-}
-
```

Comment:
Yea, I noticed that most all the `fmt` implementations of `libcore` were in the `fmt/mod.rs` file, so I moved this there as well.